### PR TITLE
Add mobile version

### PR DESCRIPTION
**Pull request by BasitKorede**
#   MY PERSONAL PORTFOLIO PROJECT MOBILE VERSION.

### This pull request adds the following.
- My recent work section.
  -  by my recent work header in h2 header.
  -  my recent work div which has the display of Grid.
   - my recent work image div. 
   - my recent work header in h3 header .
   - my recent work multiple post story (story about each project).
   - languages used while developing the projects and has the property of display flex.
   - the see project button which is assigned pseudo classes of `hover` `active` and `disabled`.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -179,6 +324,145 @@ <h2 class="about-me-header">Software Developer</h2>
           <p class="work-paragraph">Work</p>
         </div>
       </div>
+      <section class="my-recent-work">
```
Comment (adejam) [inline index.html:327]: this section or maybe another parent `div` should have the display `grid` that houses all works section. while each works container should then use a display `flex` to organize its children elements

File: index.html [hunk 2]
```diff
@@ -179,6 +324,145 @@ <h2 class="about-me-header">Software Developer</h2>
...
+            </p>
+            <ul
+              class="recent-work-ul display-flex justify-content-space-between"
+            >
+              <li class="recent-work-ul-li">css</li>
+              <li class="recent-work-ul-li">html</li>
+              <li class="recent-work-ul-li">boostrap</li>
+              <li class="recent-work-ul-li border-none">Ruby</li>
+            </ul>
+            <button
```
Comment (adejam) [inline index.html:354]: Screenshot 2021-10-07 at 06 35 32
The buttons should have the active, hover and pressed states

**Review by adejam — CHANGES_REQUESTED** [review]
### Looks good, though the works section would definitely need a revamp in a way.

Let's make changes to the review

File: index.html (hunk 2 above)
Comment (BasitKorede) [reply index.html:354]: thanks, I will make the changes now.

**Pull request merged**